portal-api/package.json

```diff
@@ -25,7 +25,7 @@
     "lint:fix": "eslint ./src --fix",
     "start": "node src/index.js",
     "start:dev": "npx nodemon src/index.js",
-    "unit-test": "jest --coverage --verbose --config=unit.jest.config.js"
+    "unit-test": "jest --verbose --config=unit.jest.config.js"
```

Comment:
Guessing this was a temporary change for local testing? I see `dtfs-central-api` has a `"unit-test-quick"` command, so maybe we could add that here too so we don't need to manually amend this when testing locally.

Reply:
Yep!